server/pxf-service/src/main/java/org/greenplum/pxf/service/PxfServiceApplication.java

```diff
@@ -1,15 +1,46 @@
 package org.greenplum.pxf.service;
 
 import org.apache.hadoop.security.PxfUserGroupInformation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+import java.net.URL;
+import java.net.URLClassLoader;
+
 /**
  * Main PXF Spring Configuration class.
  */
 @SpringBootApplication(scanBasePackages = "org.greenplum.pxf", scanBasePackageClasses = PxfUserGroupInformation.class)
 public class PxfServiceApplication {
 
+    private static final Logger LOG = LoggerFactory.getLogger(PxfServiceApplication.class);
+
+    /**
+     * Constructs a new PxfServiceApplication
+     */
+    public PxfServiceApplication() {
+        logClassLoaderInfo();
+    }
+
+    /**
+     * Logs, at info level, all the libraries loaded by the ClassLoader used by
+     * the PxfServiceApplication.
+     */
+    private void logClassLoaderInfo() {
+        ClassLoader loader = this.getClass().getClassLoader();
+        if (loader instanceof URLClassLoader) {
+            URLClassLoader urlClassLoader = (URLClassLoader) loader;
+            URL[] urls = urlClassLoader.getURLs();
+            if (urls != null) {
+                for (URL url : urls) {
+                    LOG.info("Repository {} added", url);
```

Comment:
```suggestion
                    LOG.info("Added repository {}", url);
```
since the url will be long and the word "added" after it will not be very visible

Reply:
should we say library instead? "Added library {}"

Reply:
sure, but it seems it will also list directories `Archive loaded file:/home/fguerrero/pxf-boot/conf/`, which is not a library ?

Reply:
right, I'll keep it as repository for now.